test/broker_test/broker_test.go

```diff
@@ -535,6 +535,13 @@ var _ = test.DescribeTestsFor(test.TestCase{
 								r.Status(http.StatusBadRequest).JSON().Object().Keys().NotContains("services", "credentials")
 							}, "services.0.plans.0.schemas", "{invalid")
 						})
+
+						Context("that has both supportedPlatforms and supportedPlatformNames", func() {
```

Comment:
The positive test is missing (there were two in sm-sap)

Reply:
This is already tested in the other tests of the supported platforms feature, do you think we need a dedicated one here as well?

Reply:
Yes, I thought it was appropriate to have a dedicated test, also why introduce it internally in the first place then?

Reply:
tests added